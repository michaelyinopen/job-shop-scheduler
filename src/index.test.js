import React from 'react';
import MyComponent from './index';
import renderer from 'react-test-renderer';

test('Can render demo timeline', () => {
  const component = renderer.create(
    <MyComponent />
  );
});


test('Matches snapshot', () => {
  const component = renderer.create(
    <MyComponent />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});