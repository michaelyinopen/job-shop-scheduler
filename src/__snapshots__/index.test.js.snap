// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div>
  <h2>
    Welcome to React components
  </h2>
</div>
`;
